www/base/src/app/about/about.tpl.jade

```diff
@@ -5,15 +5,14 @@
         .row
           .col-sm-4
             ul
-              li Buildbot version: {{config.bbversion}}
-              li Twisted version: {{config.txversion}}
               li Project Infos:
-                a(ng-href="{{config.projecturl}}") {{config.projectname}}
+                a(ng-href="{{config.titleURL}}") {{config.title}}
+              li(ng-repeat="v in config.versions" ng-bind-template="{{v[0]}} version: {{v[1]}}")
```

Comment:
What you say is only valid for main html page, not for templates.
Angularjs is parsing the templates before pushing them to the DOM, and the controller is also instanciated before the page is displayed, so everything is okay for using the simplified templating.